plugins/available/nvm.plugin.bash

```diff
@@ -9,13 +9,17 @@ about-plugin 'node version manager configuration'
 
 export NVM_DIR=${NVM_DIR:-$HOME/.nvm}
 # This loads nvm
-if command -v brew &>/dev/null && [ -s $(brew --prefix nvm)/nvm.sh ]
+if [ -s "$NVM_DIR/nvm.sh" ]
 then
-  . $(brew --prefix nvm)/nvm.sh
+  source "$NVM_DIR/nvm.sh"
 else
-  [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"
+  if command -v brew &>/dev/null && [ -s $(brew --prefix nvm)/nvm.sh ]
```

Comment:
While you're at it, could you please move to using `if _command_exists brew` instead of `command -v ...`? That will add logging and standardize the pattern that we're using in the plugins.